frontend/src/pages/BackOffice/BackOffice.jsx

```diff
@@ -55,7 +53,7 @@ function BackOffice() {
                 metadataMimeType: 'message/x.rsocket.routing.v0',
             },
             transport: new RSocketWebSocketClient({
-                url: 'ws://'+externalIP+':9000'
+                url: 'ws://'+externalIP+'/ws/'
```

Comment:
@ThomasVitale before merging this I wanted to know if now the `externalIP` should have the port 9000 or not? Meaning that now the env variable will be 1.1.1.1:9000 or have you removed the need for the extra port?

Reply:
Based on line 42, it looks like we still need the port.. but it will be great if we can find a way to just reuse the other netty port that is opened.

Reply:
@salaboy the port is still needed when running locally or when using a LoadBalancer service in Kubernetes, but it's not needed when using an Ingress. To support both cases, I moved the port specification to the environment variable.

Reply:
I got the app to use the same Netty for both HTTP and Websocket. I'll verify the full setup and then update the PR.